src/main/java/io/nats/service/ServiceMessage.java

```diff
@@ -41,146 +42,110 @@ public class ServiceMessage implements Message {
+     * @param headers the custom headers                 
+     */
     public void respond(Connection conn, byte[] response, Headers headers) {
         conn.publish(NatsMessage.builder().subject(message.getReplyTo()).data(response).headers(headers).build());
     }
 
+    /**
+     * Respond to a service request message with a response and custom headers.
+     * @param conn the NATS connection
+     * @param response the response payload in the form of a string
+     * @param headers the custom headers                 
+     */
     public void respond(Connection conn, String response, Headers headers) {
         conn.publish(NatsMessage.builder().subject(message.getReplyTo()).data(response).headers(headers).build());
     }
 
+    /**
+     * Respond to a service request message.
+     * @param conn the NATS connection
+     * @param response the response payload in the form of a {@link JsonSerializable} object
+     * @param headers the custom headers                 
+     */
     public void respond(Connection conn, JsonSerializable response, Headers headers) {
         conn.publish(NatsMessage.builder().subject(message.getReplyTo()).data(response.serialize()).headers(headers).build());
     }
 
-    public void respondStandardError(Connection conn, String errorMessage, int errorCode) {
+    /**
+     * Respond to a service request message with a standard error.
+     * @param conn the NATS connection
+     * @param errorText the error message text
+     * @param errorCode the error message code
+     */
+    public void respondStandardError(Connection conn, String errorText, int errorCode) {
         conn.publish(NatsMessage.builder()
             .subject(message.getReplyTo())
             .headers(new Headers()
-                .put(NATS_SERVICE_ERROR, errorMessage)
+                .put(NATS_SERVICE_ERROR, errorText)
                 .put(NATS_SERVICE_ERROR_CODE, "" + errorCode))
             .build());
     }
 
-    @Override
+    /**
+     * @return the subject that this message was sent to
+     */
     public String getSubject() {
         return message.getSubject();
     }
 
-    @Override
+    /**
+     * @return the subject the application is expected to send a reply message on
+     */
     public String getReplyTo() {
         return message.getReplyTo();
     }
 
-    @Override
+    /**
+     * @return true if there are headers
+     */
     public boolean hasHeaders() {
         return message.hasHeaders();
     }
 
-    @Override
+    /**
+     * @return the headers object for the message
+     */
     public Headers getHeaders() {
         return message.getHeaders();
     }
 
-    @Override
-    public boolean isStatusMessage() {
-        return message.isStatusMessage();
-    }
-
-    @Override
-    public Status getStatus() {
-        return message.getStatus();
-    }
-
-    @Override
+    /**
+     * @return the data from the message
+     */
     public byte[] getData() {
         return message.getData();
     }
-
```

Comment:
Glad to get rid of these unnecessary methods. Makes it clearer/simpler to use/understand.